Update README.md #20 (#22)
anitab-org/mentorship-flutter: README.md

are easy to solve.
My dream? I'll be incredibly happy, for example, if students in next year's Google Code-in will continue its development,
improve it and polish the codebase.

### Contact us

> In case of any query, feel free to reach out to the maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the mentorship system-flutter, we have a dedicated stream for this project [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/Flutter), where you can ask your doubts and interact with our community. 

Reviewer: ```suggestion
Feel free to reach out to the maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in participating in discussions related to this project, we have a dedicated stream for this project [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system), where you can ask your doubts and interact with our community. 
```
no need to mention flutter topic, that can change in the future. For now, the stream is enough.
I removed the "any query" because its also okay if they use GitHub for queries related to an issue, instead of just zulip.
